### PR TITLE
Update the expected inputs to match the output of the new symbiotic converter

### PR DESCRIPTION
**Pull request by jamartis**
TODO: the output of predator-regre-test-0003 is sometimes nondeterminic (order of the errors may differ)

### CODE REVIEW

**Comment by lzaoral** [comment]
> For example `single-c/mem-basic-realloc/0042-realloc-plain-leak-1/` on f35 (the test doesn't fail on f34 (unless I am missing something)):
> 
> ```
> diff -u ./output-exp.txt ./output-uni.txt                                                                                                                              
> --- ./output-exp.txt	2021-10-13 15:21:30.000000000 +0200
> +++ ./output-uni.txt	2021-10-13 15:21:32.000000000 +0200
> @@ -4,4 +4,4 @@
>  0042-test.c:10: note: call stack: function main ()
>  0042-test.c:10: note: Additional Info: address: 20:0
>  0042-test.c:10: note: Additional Info: pointing to: object at Segment: [20] of size 1
> -0042-test.c:10: note: Additional Info: MO9[1] allocated at main():  %4 = call noalias i8* @malloc(i64 1) #2, !dbg !15
> +0042-test.c:10: note: Additional Info: MO9[1] allocated at main():  %4 = call noalias align 16 i8* @malloc(i64 1) #2, !dbg !17
> ```

This will be fixed when I finalize the proper printing of debug info instead of LLVM IR instructions.

**Comment by lzaoral** [comment]
@jamartis The Fedora 35 Beta freeze is over so the `cppcheck` errors should be gone. Therefore, I've restarted the CI.

**Comment by lzaoral** [comment]
@jamartis Could you rebase the PR to include #21? After that the CI should be green everywhere!

**Comment by jamartis** [comment]
@lzaoral Can you please check this 40227fe commit to make sure that nothing important is being lost by it?

File: tests/single-c/mem-basic-realloc/0027-realloc-from-zerosize/output-exp@symbiotic [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (vmihalko) [inline tests/single-c/mem-basic-realloc/0027-realloc-from-zerosize/output-exp@symbiotic:1]: Maybe this shouldn't be here.

**Review by vmihalko — CHANGES_REQUESTED** [review]
LGTM - I found no change in reported errors nor in lines where errors occurred.

Change proposed in https://github.com/aufover/aufover-benchmark/pull/19#discussion_r759328881

**Review by vmihalko — APPROVED** [review]
LGTM!

**Comment by kdudka** [comment]
Thank you all!  Merging...

**Pull request merged**